deployment/goerli-deployed-contracts.json

```diff
@@ -1,23 +1,21 @@
-{
-    "USDC": "0xf17FF940864351631b1be3ac03702dEA085ba51c",
-    "EANFT": "0x026D2AED6f1E9Db160b79688Ea0CE1ABaee4Dc39",
-    "RNNFT": "0x5D34C9CA746D59a171FdDf22fD1340033c50dBd2",
-    "HumaConfig": "0x98e77059F8C9BDF3e83A49CD5F5330E5b159014c",
-    "HumaConfigTimelock": "0x0Fc7E513274cE9559d9Aa09c715DAE06b6658dFA",
-    "BaseCreditPoolTimelock": "0x6590E419B28fDFDD82fDc221b85344F7E8708197",
-    "BaseCreditPoolProxyAdminTimelock": "0xdcccF0Ee1732BE26C77d07e860AE21cBE217fcb6",
-    "BaseCreditPoolFeeManager": "0x673b3C1094AE941bb4b2eF9377DaFE3bcCc4b003",
-    "BaseCreditHDTImpl": "0x1068AE689f40B2Db2C9771e7aC64345D115DB077",
-    "BaseCreditHDT": "0x61341186E8C3B7cC0De66ae86C65943797C8Fb99",
-    "BaseCreditPoolConfig": "0x02747439954B9D29d4f0AFB0B66F3A5c0EC72850",
-    "BaseCreditPoolImpl": "0x2bD9D4BbBDf96d9645B51d6291B939F417ca33F7",
-    "BaseCreditPool": "0xA22D20FB0c9980fb96A9B0B5679C061aeAf5dDE4",
-    "ReceivableFactoringPoolTimelock": "0xb179FEaD504cFAeD016192C668FfA0bc30769840",
-    "ReceivableFactoringPoolProxyAdminTimelock": "0xdAECE171D686AcC3DA455AA6134cF45114B5b2ba",
-    "ReceivableFactoringPoolFeeManager": "0x7BA6B8eBC9b09c228582814D44D4a0F2B6B0B9E4",
-    "HDTImpl": "0x33d6526394a1C0E52293BDa8B7e9647F2266F06a",
-    "HDT": "0x27Fa332a5cA06492C2007FF4b143C921Cf779C3b",
-    "ReceivableFactoringPoolConfig": "0x01b8Ebc5DC463ee5B7abE3F8f86c9057b0e3FEa0",
-    "ReceivableFactoringPoolImpl": "0x570f47cb9F7B874A9837175b2495421f44A75536",
-    "ReceivableFactoringPool": "0x11672c0bBFF498c72BC2200f42461c0414855042"
-}
\ No newline at end of file
+{"USDC":"0xf17FF940864351631b1be3ac03702dEA085ba51c",
```

Comment:
Interesting, does your prettier format it this way? It should use the .prettierrc file in this repo... It's hard to see which addresses are changing since it doesn't keep the exact same indentation rules.

Reply:
This wasn't formatted by perttier. It was utils.js saved the file.